.gitignore

```diff
@@ -43,3 +43,9 @@ CPackSourceConfig.cmake
 
 # VSCode
 .vscode/
+/.vs
```

Comment:
I'm not sure these changes belong here.

Reply:
These changes were made because I compile in visual studio 2017, so I added the files that shouldn't sync with git,but I already removed